app/player.rb

```diff
@@ -63,6 +83,30 @@ def update
       @y = [@y, 0].max
 
       recalculate_collision_points!
+    when :inmune
+      # do the same as being alive
```

Comment:
Repeating this code sucks. One way to fix it would be to extract the movement code to a private function and call it from both `when` conditions. Something better would be to use the [State pattern](https://sourcemaking.com/design_patterns/state).